js/src/pages/price-benchmark/index.scss

```diff
@@ -1,7 +1,23 @@
 @import "@wordpress/dataviews/build-style/style.css";
 
-.gla-price-benchmark__card .app-tab-nav:first-child {
-	padding: 0 $grid-unit-40;
+.gla-price-benchmark__card {
+
+	& .app-tab-nav:first-child {
+		padding: 0 $grid-unit-40;
+	}
+
+	& .woocommerce-table.is-empty {
```

Comment:
@ankitrox Apologies i missed originally, but let's have the empty specific styles live in `js/src/pages/price-benchmark/empty-metrics-notice/index.scss`